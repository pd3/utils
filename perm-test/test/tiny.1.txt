

perm-test -t test/tiny.1.tgt.txt -b test/tiny.1.bg.txt -f test/tiny.1.fai -c test/tiny.1.calls.txt -n 1e9
perm-test -t test/tiny.1.tgt.bed -b test/tiny.1.bg.bed -f test/tiny.1.fai -c test/tiny.1.calls.bed -n 1e9

    123456789012345678901
        bbbbb  tt  bbb
               cccc
     .... .... oo.. ....
      .... .... o... ....
       .... ...o ....
        .... ..oo ....
         .... .oo. ....
    
    exact: 5./17 = 0.29411764705882354
    perm-test:     0.294090135
    err: 2.7512e-05


perm-test -t test/tiny.1.tgt.txt -b test/tiny.1.bg.txt -f test/tiny.1.fai -c test/tiny.1.calls.txt -n 1e9 --no-bg-overlap

    123456789012345678901
        bbbbb  tt  bbb
               cccc
     .... .... oo.. ....
      .... .... x... ....
       .... ...x ....
        .... ..oo ....
         .... .oo. ....
    
    exact: 3./17 = 0.17647058823529413
    perm-test:     0.1764850
    err: 1.441e-05


perm-test -t test/tiny.1.tgt.txt -b test/tiny.1.bg.txt -f test/tiny.1.fai -c test/tiny.1b.calls.txt -n 1e9
perm-test -t test/tiny.1.tgt.txt -b test/tiny.1.bg.txt -f test/tiny.1.fai -c test/tiny.1b.calls.txt -n 1e9 --no-bg-overlap

    123456789012345678901
        bbbbb  tt  bbb
               c
        .....  oo  ...
    
    exact: 2./10 = 0.2
    perm-test:     0.2
    err: 0



perm-test -t test/tiny.1.tgt.txt -b test/tiny.1.bg.txt -f test/tiny.1.fai -c test/tiny.1c.calls.txt -n 1e9

    123456789012345678901
        bbbbb  tt  bbb
               cccc
               c
    
    exact: 5./17 * 2./10 = 0.058823529411764705
    perm-test:             5.999062e-02
    err: 5.27e-06

perm-test -t test/tiny.1.tgt.txt -b test/tiny.1.bg.txt -f test/tiny.1.fai -c test/tiny.1c.calls.txt -n 1e9 --no-bg-overlap

    exact: 3./17 * 2./10 = 0.03529411764705882
    perm-test:             0.03529223
    err: 1.8876e-06


