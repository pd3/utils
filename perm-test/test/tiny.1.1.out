BG	1	5	9
TGT	1	12	13
BG	1	16	18
MSG	Maximum chromosome randomization error due to length discretization: 0.0e+00%
CALL	TGT	1	12	15
NCALLS	1	0
